deploy/crds/gcp.managed.openshift.io_projectclaims_crd.yaml

```diff
@@ -128,16 +138,18 @@ spec:
                     description: Type is the type of the condition.
                     type: string
                 required:
-                - type
                 - status
+                - type
                 type: object
               type: array
             state:
+              description: ClaimStatus is a valid value from ProjectClaim.Status
               type: string
           required:
           - conditions
           - state
           type: object
+      type: object
```

Comment:
in the past we have seen problems deploying to v3.11 with this additional `type: object`, @georgettica the solution back then was to just remove it here right?

Reply:
yup, if its the last one in the file then you.

all we need to do is deploy the CRD on a v3 cluster and see if it holds, anything else should work without a hitch

Reply:
Resolved via 488f569.